src/neuroconv/datainterfaces/ecephys/spikeglx/spikeglxconverter.py

```diff
@@ -61,28 +63,17 @@ def __init__(
         """
         folder_path = Path(folder_path)
 
-        streams = streams or self.get_streams(folder_path=folder_path)
+        streams_ids = streams or self.get_streams(folder_path=folder_path)
 
         data_interfaces = dict()
-        for stream in streams:
```

Comment:
This is the critical part of this. Previously, the Converter assumed a fix structure of the files. This PR gets rids of that assumption.